### DISCUSSION TITLE
TinyMCE version being used

### DISCUSSION THREAD

**Discussion by nasimoyz** [tinymce/tinymce-react#300] [Q&A]
Hello, thanks for all your great work on this

I need to make use of the [toolbar_sticky_offset](https://www.tiny.cloud/docs/configure/editor-appearance/#toolbar_sticky_offset) config option, but noticed its only available in tinymce v5.9 while this package uses v5.5.1.
Are there any plans to upgrade the version of tinymce used here or should I forgo this React wrapper and just consume tinymce directly?
Thanks 🙏

**Comment by nasimoyz** [comment]
False alarm, was an issue on my end

**Comment by tiny-james** [comment]
Just to clarify 5.5.1 is the minimum compatible version - specified as "^5.5.1" in the package.json. It means that the version used must be larger or equal to 5.5.1 and smaller than 6.0.0 . Once TinyMCE version 6.0.0 is released I will update the version to "^5.5.1 || ^6.0.0".

We choose that version for the package.json because it was the first that exposed the types that TinyMCE uses.

Note that it is still possible to load an even older version like TinyMCE 4.9.7 with the integration as we only call functions and read properties that existed on TinyMCE 4. It will however have incorrect types as the compiler will use the types exported by TinyMCE 5 on the interface.
